sentry/src/main/java/io/sentry/SentryTracer.java

```diff
@@ -584,17 +584,17 @@ public void setMeasurement(
 
   @Override
   public void setName(@NotNull String name) {
-    if (root.isFinished()) {
-      return;
-    }
-
-    this.name = name;
+    setName(name, TransactionNameSource.CUSTOM);
```

Comment:
Do we call this method internally?
If we do, we cannot overwrite to `custom`, only if the user does.

Reply:
Good point will check. But if so we could simply change our code to use the other overload.

Reply:
Couldn't find any direct calls to this overload (outside of tests).

Reply:
Should we check cross platform SDKs before merging this?

Reply:
@marandaneto should we check any SDKs first or can I go ahead and merge this PR?

Reply:
LGTM